packages/legacy-scripting-runner/test/integration-test.js

```diff
@@ -2068,6 +2091,8 @@ describe('Integration Test', () => {
           should.equal(content.args.foo, '1');
           should.equal(content.args.bar, 'hello');
           should.not.exist(content.headers['X-Api-Key']);
+          should.equal(output.input.bundle.request.headers.Accept, 'application/json');
+          should.not.exist(output.input.bundle.request.headers['Content-Type']);
```

Comment:
These two lines are unnecessary because for file hydrators, the request options are built up [from scratch](https://github.com/zapier/zapier-platform/blob/7d200799531b0ae13685ca0ed01464d1583b1cbb/packages/legacy-scripting-runner/index.js#L987-L988). That is, file hydrators aren't affected by the initial request headers you added [here](https://github.com/zapier/zapier-platform/blob/00c64d99a370895d1d77e7d07b77f0bb94bed80c/packages/legacy-scripting-runner/index.js#L1001-L1002).